zanata-war/src/main/java/org/zanata/service/impl/TranslationMemoryServiceImpl.java

```diff
@@ -90,6 +101,27 @@
     private static final int SEARCH_MAX_RESULTS = SysProperties.getInt(
             SysProperties.TM_MAX_RESULTS, 20);
 
+    private static final float BOOST_CONTENT = SysProperties.getFloat(
+            SysProperties.TM_BOOST_CONTENT, 10.0f);
+
+    private static final float BOOST_TFTID = SysProperties.getFloat(
+            SysProperties.TM_BOOST_TFTID, 10.0f);
+
+    private static final float BOOST_PROJECT = SysProperties.getFloat(
+            SysProperties.TM_BOOST_PROJECT, 2.0f);
+
+    private static final float BOOST_DOCID = SysProperties.getFloat(
+            SysProperties.TM_BOOST_DOCID, 1.5f);
+
+    private static final float BOOST_RESID = SysProperties.getFloat(
+            SysProperties.TM_BOOST_RESID, 1.5f);
+
+    private static final float BOOST_ITERATION = SysProperties.getFloat(
+            SysProperties.TM_BOOST_ITERATION, 1.0f);
```

Comment:
> Does this mean that we can now configure the boosting properties via System props? If so, do we want to document it?

@carlosmunoz It should be configurable now, in emergencies, but I don't think we want to make it a supported feature.  You shouldn't be messing with these boost values unless you have read this source code, or under advice from someone who has.